### ISSUE TITLE
cURL error: (60) ... certificate not valid

### ISSUE THREAD

**Issue by stefanocurnis** [sh0umik/Laravel5-PaypalIPN#13]
Hey guys. I suddenly get this in my IPN responses:

cURL error: [60] SSL certificate problem: unable to get local issuer certificate.

Anyone else?
I never seen it before. Probably outdated certificate file?
Thanks

**Comment by helderferrari2** [comment]
Same error here... any solution?

**Comment by ultrono** [comment]
@helderferrari2 @stefanocurnis Yep, have suddenly started to get this myself recently too. Nothing has changed on the server.

I resolved this by saving https://curl.haxx.se/ca/cacert.pem into the src/cert directory. 

I then replaced https://github.com/sh0umik/Laravel5-PaypalIPN/blob/master/src/PaypalIPNListener.php#L106 with

```
dirname(__FILE__)."/cert/cacert.pem");
```

I'm unsure why this worked :/

Above works in the Sandbox and live environments.

**Comment by ultrono** [comment]
FYI the offical PayPal IPN examples on Github (see https://github.com/paypal/ipn-code-samples/blob/master/php/PaypalIPN.php) use the cacert.pem I mentioned above and differrent endpoints. IT may be wise to use that instead of this class.

**Comment by 8ctopus** [comment]
#16
